src/compiler/crystal/codegen/llvm_typer.cr

```diff
@@ -462,15 +462,15 @@ module Crystal
     end
 
     # Copy existing LLVM types, possibly from another context,
-    # into this typer's context.
+    # into this type's context.
```

Comment:
I think this is not a typo. Because this file name is <code>llvm_type<b>r</b>.cr</code> and this comment says about the **typer**.